scripts/dts/gen_defines.py

```diff
@@ -36,7 +36,8 @@ def main():
         edt = edtlib.EDT(args.dts, args.bindings_dirs,
                          # Suppress this warning if it's suppressed in dtc
                          warn_reg_unit_address_mismatch=
-                             "-Wno-simple_bus_reg" not in args.dtc_flags)
+                             "-Wno-simple_bus_reg" not in args.dtc_flags,
+                         default_prop_types=False)
```

Comment:
Don't you want to remove default_prop_types here.

Reply:
It becomes True later on in the series (`devicetree: allow access to all nodes`). I'm just trying to preserve bisectability of behavior.